aws-lc-rs/tests/data/ecdsa_sign_fixed_tests.txt

```diff
@@ -250,6 +250,129 @@ Q = 04a39ac353ca787982c577aff1e8601ce192aa90fd0de4c0ed627f66a8b6f02ae51315543f72
+Sig = f5e4a98b9b29b28e7f54cf594513ae2285b057c7375f8c46f2232188d3f532dd6ebf9f640c116c60fda5ee5ffeef91a9c0bf4df7e388a417b942bda51ebb27e3ea31601cce9590a5e45bf3e0c6105bc8afd44271dabbe5e033a77ca9ecfa6f20
+
+Curve = P-384
+Digest = SHA3-384
+Msg = d9b5cf0b50416573ff3c63133275a18394dd4326be2041e8d97e6e4e3855a4a177e9d26dfd223fe8aa74564edb49bd72de19916fb6f001f44530d5c18e2c332bce1b7415df5927ece5f3824f34d174b963136b53aef1fb78fb0c06a201a40b2db38e4d8216fc1e392a798c8ab4b3a314496b7f1087804ebfa89bf96e9cdb80c0
+d = 083e7152734adf342520ae377087a223688de2899b10cfcb34a0b36bca500a4dfa530e2343e6a39da7ae1eb0862b4a0d
+Q = 0470a0f16b6c61172659b027ed19b18fd8f57bd28dc0501f207bd6b0bb065b5671cf3dd1ed13d388dcf6ccc766597aa6044f845bf01c3c3f6126a7368c3454f51425801ee0b72e63fb6799b4420bfdebe3e37c7246db627cc82c09654979c700bb
+k = 28096ababe29a075fbdf894709a20d0fdedb01ed3eeacb642a33a0da6aed726e13caf6cf206792ec359f0c9f9b567552
+Sig = c65586f16ebf6b4819a1201353d96b3512baad68f9114bb60dcf3d5b26ff3f70a677d3e40a5df38375c1c91dd26b2f00ceee0d4a82212da65b7ea1b861f675dd5aac7c8e6b64f91d6ac0c7507e3599901bd8bc7d1856c5826c836d94127a29aa
+
+Curve = P-384
+Digest = SHA3-384
+Msg = 9e4042d8438a405475b7dab1cd783eb6ce1d1bffa46ac9dfda622b23ac31057b922eced8e2ed7b3241efeafd7c9ab372bf16230f7134647f2956fb793989d3c885a5ae064e85ed971b64f5f561e7ddb79d49aa6ebe727c671c67879b794554c04de0e05d68264855745ef3c9567bd646d5c5f8728b797c181b6b6a876e167663
+d = 63578d416215aff2cc78f9b926d4c7740a77c142944e104aa7422b19a616898262d46a8a942d5e8d5db135ee8b09a368
+Q = 04cadbacef4406099316db2ce3206adc636c2bb0a835847ed7941efb02862472f3150338f13f4860d47f39b7e098f0a390752ad0f22c9c264336cde11bbc95d1816ed4d1b1500db6b8dce259a42832e613c31178c2c7995206a62e201ba108f570
+k = 7b69c5d5b4d05c9950dc94c27d58403b4c52c004b80a80418ad3a89aabc5d34f21926729e76afd280cc8ee88c9805a2a
+Sig = bc8702e5d848d957041e5eb28ee2b6bd83d2f3efd8f3762339b1ed739ef09e77a4bd08a20d9ac77cb25cc7c310f4cd83a920d40c1701fee3ed395308b6bb46e71274c145fcdf9effee3d435e30fbc95e30edbe618e5c07c427d48d7d58698bdf
+
+Curve = P-384
+Digest = SHA3-384
+Msg = 0b14a7484a40b68a3ce1273b8a48b8fdb65ba900d98541c4bbd07b97e31bcc4c85545a03e9deab3c563f47a036ff60d0361684ba241b5aa68bb46f440da22181ee328a011de98eff34ba235ec10612b07bdfa6b3dc4ccc5e82d3a8d057e1862fef3def5a1804696f84699fda2ec4175a54a4d08bcb4f0406fdac4eddadf5e29b
+d = ed4df19971658b74868800b3b81bc877807743b25c65740f1d6377542afe2c6427612c840ada31a8eb794718f37c7283
+Q = 0433093a0568757e8b58df5b72ea5fe5bf26e6f7aeb541b4c6a8c189c93721749bcaceccf2982a2f0702586a9f812fc66febe320d09e1f0662189d50b85a20403b821ac0d000afdbf66a0a33f304726c69e354d81c50b94ba3a5250efc31319cd1
+k = d9b4cd1bdfa83e608289634dbfcee643f07315baf743fc91922880b55a2feda3b38ddf6040d3ba10985cd1285fc690d5
+Sig = 6a6e563d2ad297b79391795adf97ec33a5dcf5f65256914c255c02f03e91150d4d2d9ce804cc5e54015a7882d3e6ab72c2b04d184ab4a685a12f370b30ca2081c51b2bf731420befa8322926f2ed3ac72c67abdc66e9f16514407183586f252e
+
+Curve = P-384
```

Comment:
Seems like there is more extensive testing of the fixed signature length, then the asn1 encoding? Are these just testing variable length signature and verifying the expected padding length?

Reply:
Like the other file, these test vectors [originated from ring](https://github.com/briansmith/ring/blob/95948b3977013aed16db92ae32e6b8384496a740/src/ec/suite_b/ecdsa/ecdsa_sign_fixed_tests.txt), which has many more test vectors covering "fixed" than "asn1". 

I'm not sure our library requires as extensive coverage of test cases since we defer cryptographic operations to AWS-LC (whereas ring maintains its own cryptographic operations).